src/MetricsUploader/include/MetricsUploader/MetricsUploader.h

```diff
@@ -49,7 +49,8 @@ class MetricsUploader {
   // Returns true on success and false otherwise.
   virtual bool SendLogEvent(
       OrbitLogEvent_LogEventType log_event_type,
-      std::chrono::milliseconds event_duration = std::chrono::milliseconds::zero()) = 0;
+      std::chrono::milliseconds event_duration = std::chrono::milliseconds::zero(),
+      OrbitLogEvent_StatusCode status_code = OrbitLogEvent_StatusCode_SUCCESS) = 0;
```

Comment:
Default arguments for virtual function are not ideal, and shouldn't be used according to the style guide: https://google.github.io/styleguide/cppguide.html#Default_Arguments

Reply:
Sorry, for raising this now. This probably slipped through one of the previous reviews.

Reply:
No problem at all. Better late than never. I wrote a comment in the main thread and changed some things. ptal